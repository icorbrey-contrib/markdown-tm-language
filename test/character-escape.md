a \- b.

c \! d.

e \\ f.

g \a h.

i \\\ j.

k \ l.
