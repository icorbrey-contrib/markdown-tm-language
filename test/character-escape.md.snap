>a \- b.
#^^ text.md meta.paragraph.md
#  ^^ text.md meta.paragraph.md constant.language.character-escape.md
#    ^^^^ text.md meta.paragraph.md
>
>c \! d.
#^^ text.md meta.paragraph.md
#  ^^ text.md meta.paragraph.md constant.language.character-escape.md
#    ^^^^ text.md meta.paragraph.md
>
>e \\ f.
#^^ text.md meta.paragraph.md
#  ^^ text.md meta.paragraph.md constant.language.character-escape.md
#    ^^^^ text.md meta.paragraph.md
>
>g \a h.
#^^^^^^^^ text.md meta.paragraph.md
>
>i \\\ j.
#^^ text.md meta.paragraph.md
#  ^^ text.md meta.paragraph.md constant.language.character-escape.md
#    ^^^^^ text.md meta.paragraph.md
>
>k \ l.
#^^^^^^^ text.md meta.paragraph.md
>